.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,24 @@
+# Need help? 
+Before open a new issue, did you ask your question on
+* [Google group](http://groups.google.com/group/testng-users)
+* [StackOverflow](http://stackoverflow.com/questions/tagged/testng)
+
+If you double post on both site, please provides the link to the other question to avoid duplicate answer.
+
+# Are you sure it is a TestNG bug?
+Before posting the issue, try to reproduce the issue with the CLI.
+
+If the problem does not exist with the CLI, first check if the issue exists on the bugtracker of the runner, and open an issue there first:
+* Eclipse: https://github.com/cbeust/testng-eclipse/issues
+* IntelliJ: [https://youtrack.jetbrains.com/issues](https://youtrack.jetbrains.com/issues?q=%23%7BUnit%20Testing.%20TestNG%7D%20)
+* Maven: https://issues.apache.org/jira/browse/SUREFIRE
+* Gradle: https://issues.gradle.org/projects/GRADLE
+
+# Which version are you using?
+Up to now, only the latest version is supported. Try to reproduce the issue with the latest version.
```

Comment:
Not sure how to read this, it seems to self contradict. I suggest:

Always make sure your issue is happening on the latest TestNG version. Bug reports occurring on older versions will not be looked at.